### PR TITLE
add function calcIDataElecSteamGen and iDataElecSteamGen.csv to fullOPEN-PROM

### PR DESCRIPTION
**Pull request by Fsioutas**
add function calcIDataElecSteamGen and iDataElecSteamGen.csv to fullOPEN-PROM

### CODE REVIEW

File: R/fullOPEN-PROM.R [hunk 1]
```diff
@@ -148,7 +148,35 @@ fullOPEN_PROM <- function() {
               sep = ",",
               col.names = FALSE,
               append = TRUE)
-
+  
+  x <- calcOutput("IDataElecSteamGen", aggregate = FALSE)
```
Comment (giannou) [inline R/fullOPEN-PROM.R:152]: why not `aggregate = TRUE`? capacities are extensive variables

**Review by giannou — CHANGES_REQUESTED** [review]

**Comment by giannou** [comment]
@derevirn for this PR to be further processed a confirmation is needed that the dataset is created in the correct OPEN-PROM format, and with the correct values for the OPEN-PROM countries

**Comment by derevirn** [comment]
> @derevirn for this PR to be further processed a confirmation is needed that the dataset is created in the correct OPEN-PROM format, and with the correct values for the OPEN-PROM countries

@giannou The dataset complies to the OPEN-PROM dataset format. Also, I have compared the total capacity values to MENA-EDS and ENERDATA, and they are correct.

File: R/fullOPEN-PROM.R (hunk 1 above)
Comment (derevirn) [reply R/fullOPEN-PROM.R]: @giannou We discussed this with @Fsioutas , and he made another commit where `aggregate = TRUE`. Review the changes when possible.

**Review by giannou — APPROVED** [review]

**Pull request merged**